enterprise/core-edge/src/test/java/org/neo4j/coreedge/scenarios/CoreToCoreCopySnapshotIT.java

```diff
@@ -251,4 +253,9 @@ private String string( int numberOfCharacters )
         }
         return s.toString();
     }
+
+    private CoreState getCoreState( CoreGraphDatabase follower )
+    {
+        return follower.getDependencyResolver().resolveDependency( CoreState.class );
```

Comment:
I consider the SPI-based version nicer because you don't have to resolve arbitrary classes and access them sideways by resolving. It makes all the features explicitly exposed.